index.html

```diff
@@ -1,17 +1,18 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>INFO8002: Large-scale database systems</title>
+    <title>INFO8002: Large-scale Database Systems</title>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
     <link rel="stylesheet" href="./assets/style.css">
     <link rel="stylesheet" href="./assets/grid.css">
+    <link rel="stylesheet" href="./assets/custom.css">
   </head>
   <body>
     <textarea id="source"></textarea>
     <script src="./assets/remark-latest.min.js"></script>
     <script src="./assets/auto-render.min.js"></script>
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.8.3/katex.min.css" integrity="sha384-B41nY7vEWuDrE9Mr+J2nBL0Liu+nl/rBXTdpQal730oTHdlrlXHzYMOhDU60cwde" crossorigin="anonymous">
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.8.3/katex.min.js" integrity="sha384-L9gv4ooDLrYwW0QCM6zY3EKSSPrsuUncpx26+erN0pJX4wv1B1FzVW1SvpcJPx/8" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./assets/katex.min.css">
+    <script src="./assets/katex.min.js"></script>
```

Comment:
I had done this before, but this triggers JS errors because it cannot find the font files. If we ship katex locally, the font files should come as well.

Reply:
Aaaahhhh, I have a solution for that! I use KaTeX on my website as well, give me a moment!